frontend/src/component/providers/AccessProvider/permissions.ts

```diff
@@ -36,3 +36,6 @@ export const DELETE_SEGMENT = 'DELETE_SEGMENT';
 export const APPLY_CHANGE_REQUEST = 'APPLY_CHANGE_REQUEST';
 export const APPROVE_CHANGE_REQUEST = 'APPROVE_CHANGE_REQUEST';
 export const SKIP_CHANGE_REQUEST = 'SKIP_CHANGE_REQUEST';
+export const READ_USER_PAT = 'READ_USER_PAT';
```

Comment:
Do we need READ_USER_PAT? If the user owns the PAT, then they should be the only ones having access to it. Now if we think about service accounts, probably it's not a Personal Access Token but a Service Access Token (implementation-wise they'll be pretty similar, but regarding permissions, it may imply that different "users" or services can access the token). 

Just some random thoughts, let me know what you think.

Reply:
I feel like for all intents and purposes they are still PATs, whether they belong to a user or a service account. The idea is that there's not really much of a difference between regular users and SAs. If we are introducing other-users-PAT-management then I think it makes sense that we should have a permission that controls whether we are able to read them.

Reminder that reading/fetching PATs, whether our own or belonging to other users, never return the secret (PAT itself), only the auxiliary info.

Reply:
True, reading a service account PAT would be a good use case for a user reading another ~user~ service account PAT